tests/vmi_networking_test.go

```diff
@@ -791,6 +793,117 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 				}, 120*time.Second).Should(Succeed())
 			})
 		})
+
+		Context("MTU verification", func() {
```

Comment:
Unless  missed something, this test is using the default MTU which should have no issues even today. The problem occurs when the specified MTU is bigger than the default dummy one (1500?).

However, I am not sure if you can set a bigger mtu (we may need to adjust/check kubevirtci for this).

Reply:
Yes, the test is using the mtu the cni provides. kubevirt doesn't have api to control the mtu.
So yes, if we want jumbo frames test we need to whether update cni on our existing cluster or create a new cluster.

However, the reason to the bug is the bridge got its mtu from k6t-eth0-nic and not from the tap device.
The existing test verifies the k6t-eth0 bridge, k6t-eth0-nic and the vnic have the same correct mtu. Due to the urgency, I think is the best we can have atm. The test will fail on existing cluster without the fix in the first commit of the PR (so it passes the "make sure to start with a failing test" criteria:)).